### PR TITLE
new review and error fixing

### PR DESCRIPTION
**Pull request by joel-powell**
New features:
- New review page
- Nav search component
- WithNav component with dummy nav bar and content wrapper

Changes:
- getPropertyByAddress controller now accepts exact address or terms
- changed favicon to tenant talk logo
- changed site title to Tenant Talk
- updated home screen search prompt text
- fixed capitalisation on LoginForm component
- renamed results page to properties
- renamed PropertyList to PropertyCard and moved mapping logic to Properties component
- fixed capitalisation on PropertyReviews
- fixed capitalisation on ReviewCard, de-structured and validated props
- encapsulated stars to new component
- various other error fixing including adding proptypes where needed

### CODE REVIEW

**Review by mirandaweston — DISMISSED** [review]
🍾

File: frontend/tailwind.config.js [hunk 1]
```diff
@@ -4,8 +4,5 @@ module.exports = {
   theme: {
     extend: {},
   },
-  plugins: [
-    require("@tailwindcss/forms"),
-    require('@tailwindcss/typography')
-  ],
+  plugins: [require("@tailwindcss/forms")],
```
Comment (iamdobbs) [inline frontend/tailwind.config.js:7]: Do we no longer need the typography plugin?

**Review by iamdobbs — CHANGES_REQUESTED** [review]

File: frontend/tailwind.config.js (hunk 1 above)
Comment (joel-powell) [reply frontend/tailwind.config.js:7]: Didn't seem to be in use by anything and was throwing loads of errors 😅

**Review by iamdobbs — APPROVED** [review]

**Review by mirandaweston — APPROVED** [review]
🥇

**Pull request merged**